pod/recorder/views.py

```diff
@@ -221,6 +221,67 @@ def reformat_url_if_use_cas_or_shib(request, link_url):
 """
 
 
+def video_publish(recorder, mediapath, request, course_title):
+    if recorder.publication_auto is True:
+        # Ici on a déjà controllé que l'upload est terminé
+        rft = RecordingFileTreatment.objects.get(file=mediapath, recorder=recorder)
+        title = rft.file.split("/")
+        title = title[len(title) - 1]
+        recording = Recording(
+            title=title,
+            type=rft.type,
+            source_file=rft.file,
+            user=recorder.user,
+            recorder=rft.recorder,
+        )
+        recording.save()
+        rft.delete()
+        return HttpResponse("auto")
+    else:
+        link_url = "".join(
+            [
+                request.build_absolute_uri(reverse("record:add_recording")),
+                "?mediapath=",
+                mediapath,
+                "&course_title=%s" % course_title,
+                "&recorder=%s" % recorder.id,
+            ]
+        )
+        # link_url = reformat_url_if_use_cas_or_shib(request, link_url)
+
+        html_msg = _(
+            "<p>Hello,<br>a new recording has just be added on %("
+            "title_site)s from the recorder “%(recorder)s”.<br>"
+            'To add it, just click on link below.</p><p><a href="%('
```

Comment:
Il y a répétition du verbe "add" ("has just been added", puis "to add it").
Pour plus de clarté, préciser ce que fait l'action de l'utilisateur.

Reply:
Il faut que je regarde à quoi correspond ce texte. Si tu sais quoi mettre ; je fais la modife tout de suite

Reply:
C'est le contenu du mail pour la partie semie auto ; ok je regarde

Reply:
"To assign it" à la place de "To add it"; ça te parait plus pertinent ?

Reply:
YES ;) Merci, je trouvais pas quoi mettre justement ^^